public/index.html

```diff
@@ -40,4 +40,17 @@
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
   </body>
+  <script>
```

Comment:
You should also:

1) Encapsulate the script that's executed twice in a function and call that instead of writing it out twice (DRY)
2) Fix indentation
3) Fix nesting - script tags go in the body, not in html